*** COLLADABaseUtils/include/COLLADABUPcreCompiledPattern.h.orig	2020-05-15 10:26:03.000000000 +0100
--- COLLADABaseUtils/include/COLLADABUPcreCompiledPattern.h	2020-05-15 10:26:46.000000000 +0100
***************
*** 13,20 ****
  
  #include "COLLADABUPrerequisites.h"
  
! struct real_pcre;
! typedef struct real_pcre pcre;
  
  
  namespace COLLADABU
--- 13,19 ----
  
  #include "COLLADABUPrerequisites.h"
  
! #include <pcre.h>
  
  
  namespace COLLADABU
