*** pom.xml.orig	2020-05-06 23:09:07.000000000 +0100
--- pom.xml	2020-05-07 01:08:50.000000000 +0100
***************
*** 85,91 ****
  		<binary-revision>102</binary-revision>
  
  		<maven-javadoc-plugin-version>3.2.0</maven-javadoc-plugin-version>
! 		<git-commit-id-plugin-version>2.2.4</git-commit-id-plugin-version>
  		<maven-assembly-plugin-version>3.2.0</maven-assembly-plugin-version>
  		<maven-enforcer-plugin-version>3.0.0-M3</maven-enforcer-plugin-version>
  
--- 85,91 ----
  		<binary-revision>102</binary-revision>
  
  		<maven-javadoc-plugin-version>3.2.0</maven-javadoc-plugin-version>
! 		<!-- <git-commit-id-plugin-version>2.2.4</git-commit-id-plugin-version> -->
  		<maven-assembly-plugin-version>3.2.0</maven-assembly-plugin-version>
  		<maven-enforcer-plugin-version>3.0.0-M3</maven-enforcer-plugin-version>
  
***************
*** 750,756 ****
  										<ignore></ignore>
  									</action>
  								</pluginExecution>
! 								<pluginExecution>
  									<pluginExecutionFilter>
  										<groupId>
  											pl.project13.maven
--- 750,756 ----
  										<ignore></ignore>
  									</action>
  								</pluginExecution>
! 								<!-- <pluginExecution>
  									<pluginExecutionFilter>
  										<groupId>
  											pl.project13.maven
***************
*** 768,774 ****
  									<action>
  										<ignore></ignore>
  									</action>
! 								</pluginExecution>
  								<pluginExecution>
  									<pluginExecutionFilter>
  										<groupId>
--- 768,774 ----
  									<action>
  										<ignore></ignore>
  									</action>
! 								</pluginExecution> -->
  								<pluginExecution>
  									<pluginExecutionFilter>
  										<groupId>
***************
*** 848,854 ****
  				These are used in the project.properties files.
  				See: https://github.com/ktoso/maven-git-commit-id-plugin
  			-->
! 			<plugin>
  				<groupId>pl.project13.maven</groupId>
  				<artifactId>git-commit-id-plugin</artifactId>
  				<version>${git-commit-id-plugin-version}</version>
--- 848,854 ----
  				These are used in the project.properties files.
  				See: https://github.com/ktoso/maven-git-commit-id-plugin
  			-->
! 			<!-- <plugin>
  				<groupId>pl.project13.maven</groupId>
  				<artifactId>git-commit-id-plugin</artifactId>
  				<version>${git-commit-id-plugin-version}</version>
***************
*** 865,871 ****
  					<verbose>false</verbose>
  					<generateGitPropertiesFile>false</generateGitPropertiesFile>
  				</configuration>
! 			</plugin>
  
  			<plugin>
  				<artifactId>maven-compiler-plugin</artifactId>
--- 865,871 ----
  					<verbose>false</verbose>
  					<generateGitPropertiesFile>false</generateGitPropertiesFile>
  				</configuration>
! 			</plugin> -->
  
  			<plugin>
  				<artifactId>maven-compiler-plugin</artifactId>
